react-ui/src/pages/submission/SubmissionForm.jsx

```diff
@@ -172,13 +178,23 @@ function SubmissionForm({
       let errorValue = null;
       switch (field) {
         case 'category':
-          if (!categories.length) {
-            break;
+          if (categories.length && !formState[field].value) {
+            errorValue = 'This is a required question. ';
           }
-        // Fall through
+          break;
         case 'description':
+          if (!formState[field].value) {
```

Comment:
_Optional Nit_: This is from previous code, so you don't have to change it, but it may be nice to extract all the `formState[field].value` usages to a constant.